### DISCUSSION TITLE
Drag bugs

### DISCUSSION THREAD

**Discussion by Dimicrock** [ppy/osu#22641] [Q&A]
The cursor does not move when I touch the tablet with the pen

**Comment by Axilotl17** [comment]
please include logs & tablet model.

are you using lazer's tablet driver?

**Reply by Dimicrock** [reply]
I use huion driver

**Reply by Axilotl17** [reply]
please include logs. and try using lazer's built-in tablet driver.

**Comment by MishterKirby** [comment]
Depending on the model, you might have to disable the "High precision cursor" setting, since it conflicts with some tablets
